kafka/Dockerfile

```diff
@@ -1,34 +1,49 @@
-# syntax=docker/dockerfile:1.10.0@sha256:865e5dd094beca432e8c0a1d5e1c465db5f998dca4e439981029b3b81fb39ed5
-# check=error=true
```

Comment:
These changes and a few others were accidentally introduced by me because I messed up the merging when updating this branch to the current state of the main branch. Fixed it.